### DISCUSSION TITLE
Can I use parcel only for compiling scss / sass files?

### DISCUSSION THREAD

**Discussion by makkabi** [parcel-bundler/parcel#4965] [General]
I want to use a .scss file as entry point, and generate a css file. For example like this:
`parcel watch scss/style.scss --out-dir css`

This generates css/style.css (and style.css.map, which is fine), but it also generates style.js and style.js.map, which I don't want.

Since there are, as far as I know, still no CLI tools for really easily (!) compile scss with basic postcss plugins (Autoprefixer, cssnano),  I think this would make parcel really useful for a whole new group of users who only need the scss part, and it could be the default solution for e.g. a lot of tutorials.

**Comment by mischnic** [comment]
> parcel watch scss/style.scss --out-dir css

add `--no-hmr` ?

**Reply by makkabi** [reply]
Thanks, but sorry, this still generates the JS files.

**Reply by andremendonca** [reply]
for anyone getting into this thread from google like myself, it does work. There is another thread related #711
